testing-overhead/src/test/java/io/opentelemetry/containers/PetClinicRestContainer.java

```diff
@@ -54,6 +53,14 @@ public PetClinicRestContainer(Network network, Startable collector, Agent agent,
         .withExposedPorts(PETCLINIC_PORT)
         .withFileSystemBind(namingConventions.localResults(), namingConventions.containerResults())
         .waitingFor(Wait.forHttp("/petclinic/actuator/health").forPort(PETCLINIC_PORT))
+        .withEnv("spring_profiles_active", "hsqldb,spring-data-jpa")
+        .withEnv("spring_datasource_url", "jdbc:postgresql://postgres:5432/" + PostgresContainer.DATABASE_NAME)
+        .withEnv("spring_datasource_username", PostgresContainer.USERNAME)
+        .withEnv("spring_datasource_password", PostgresContainer.PASSWORD)
+        .withEnv("spring_datasource_driver-class-name", "org.postgresql.Driver")
+        .withEnv("spring_jpa_database", "POSTGRESQL")
+        .withEnv("spring_jpa_database-platform", "org.hibernate.dialect.PostgreSQLDialect")
```

Comment:
This certainly should not be needed.

Reply:
Right, it figures out a sane default, and I confirmed it.

Reply:
Doh, sorry I didn't recognize the multi-line. I removed the other two and it still works as you predicted.  :) Thanks.